server/providers/pouchdb.js

```diff
@@ -298,6 +298,48 @@ const pouchDb = (app, io, pouchOptions) => {
     ).catch(() => res.send([]))
   })
 
+  app.get('/:wargame/lastDoc/:id?', async (req, res) => {
+    const databaseName = checkSqliteExists(req.params.wargame)
+
```

Comment:
@lilitkarapetyan - in the above call to `checkSqliteExists`, shouldn't we use the `:wargame` from the end-point?